sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml

```diff
@@ -93,7 +93,7 @@ extensions:
       title: "origin prerequisites"
       repository: "aos-cd-jobs"
       script: |-
-        crio_tag="$( cat ./ORIGIN_PKG_VERSION | grep -Po '\d.\d' )"
+        crio_tag="$( cat ./ORIGIN_PKG_VERSION | grep -Po '\d\.\d' | head -1 )"
```

Comment:
You don't need Perl regex for this

Reply:
And in what case does `ORIGIN_PKG_VERSION` have more than one line?

Reply:
never, but if you get "-3.9.0-something3.4-something432" you'll get:
3.9
3.4

so, that's why the `head`

Reply:
That's what's happening in https://github.com/openshift/openshift-ansible/pull/6449 for instance
```
-3.9.0-0.alpha.0.169.a2ec324

$ echo "-3.9.0-0.alpha.0.169.a2ec324" | grep -Po '\d\.\d'        1 ↵
3.9
0.1 
```

Reply:
> You don't need Perl regex for this

do you have any suggestion about doing this w/o perl regexp? I'll happly use your command

Reply:
The current regex can be written without Perl syntax, but we can be more specific anyway. It is fair to assume that we could do:

```sh
crio_tag="$( grep -Po "(?<=^-)[0-9]+\.[0-9]+" ./ORIGIN_PKG_VERSION )"
```

Reply:
fixed, thanks a ton @stevekuznetsov